graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java

```diff
@@ -528,9 +526,35 @@ public GraphQLObjectType buildType(Table table) {
       }
     }
 
+    buildAggregationFunctions(builder);
```

Comment:
I assume that we want to add a possibility for all supported functions for all tables.